thrift_src/standard.thrift

```diff
@@ -231,6 +231,18 @@ exception InvalidCrcOperation {
  1:CrcErrorCode code
 }
 
+typedef list<i8> BmToeplitzRssKey
+
+enum ToeplitzRssHashErrorCode {
```

Comment:
for consistency, everything in the Thrift service should use the `ToeplitzRssHash` / `toeplitz_rss_hash` prefix

Reply:
although I'm thinking more about this, and I don't see why we use "rss" everywhere. RSS is just one use case for a hash function, and in bmv2 there is nothing tying this hash function to RSS.

So IMO, we should use `toeplitz_hash_` / `ToeplitzHash` for the Thrift service, and `toeplitz_` / `Toeplitz` for the bm_sim core library.

Reply:
Sure, I'll adopt this naming scheme.